plugin.yaml

```diff
@@ -40,6 +40,22 @@ node_types:
                 A dictionary of parameters allowed by docker-py to the
                 create_container function.
               default: {}
+            daemon_client:
+              description: >
+              Docker client information necessary to connect to Docker service
+                - transport_protocol
+                - localhost
+                - host_port
+                - api_version
+                - connection_timeout
+                - enable_tls
+              default:
+                transport_protocol: tcp
+                host_string: localhost
+                host_port: 2375
+                api_version: '1.19'
+                connection_timeout: 60
+                enable_tls: false
```

Comment:
You're talking about Cloudify 3.3, but for vCloud Blueprinting service it is necessary to align to stable version which is 3.2